### PR TITLE
fix: auto-select the sole visible workflow

### PR DESCRIPTION
**Pull request by carlosflorencio**
A hidden system workflow could make the New Task dialog treat a workspace with one usable workflow as having multiple choices. The dialog now selects the sole visible workflow implicitly and removes the redundant selector.

## Validation

- `cd apps && pnpm --filter @kandev/web test -- --run components/task-create-dialog-defaults.test.ts`
- `cd apps/web && pnpm run typecheck`
- `cd apps/web && pnpm e2e:run --no-build tests/task/create-task.spec.ts -- --grep "selects the single visible workflow"`
- Captured and visually inspected fresh desktop and phone-width dialog screenshots using synthetic, disposable local data.
- Reviewed `docs/public/**`; this internal dialog simplification requires no public documentation update.

## Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have manually tested my changes and they work as expected.
- [ ] My changes have tests that cover the new functionality and edge cases.
- [ ] If my change touches UI files (`apps/web/`), I have added or updated Playwright e2e tests in `apps/web/e2e/` and verified them with `make test-e2e`.
- [ ] I checked whether this affects public docs in `docs/public/**` and updated them or noted why no docs change is needed.

<a href="https://cubic.dev/pr/kdlbs/kandev/pull/2020?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

### CODE REVIEW

File: apps/web/e2e/tests/task/create-task.spec.ts [hunk 1]
```diff
@@ -12,6 +12,25 @@ const START_ENABLED_TIMEOUT = 30_000;
...
+      workspace_id: seedData.workspaceId,
+      workflow_filter_id: "",
+    });
+
+    const kanban = new KanbanPage(testPage);
+    await kanban.goto();
+    await kanban.createTaskButton.first().click();
+
+    await expect(testPage.getByTestId("create-task-dialog")).toBeVisible();
+    await expect(testPage.getByTestId("workflow-selector-trigger")).toHaveCount(0);
```
[the remark this answers is not in the document]
Comment (carlosflorencio) [reply apps/web/e2e/tests/task/create-task.spec.ts]: Scoped the selector assertion to the active create-task dialog and reran the focused Chromium E2E test successfully.
